src/ApiService/ApiService/onefuzzlib/notifications/JinjaTemplateAdapter.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.OneFuzz.Service;
+
+public class JinjaTemplateAdapter {
```

Comment:
Will this convert every jinja template to scriban or is it a best effort implementation ?

Reply:
In the scope of all jinja templates, it's definitely best effort. In the scope of the jinja templates we use, it will convert all of them.